modules/end-user-guide/partials/proc_setting-up-che-editor-yaml.adoc

```diff
@@ -1,16 +1,14 @@
 [id="setting-up-che-editor-yaml"]
 = Setting up che-editor.yaml
 
-To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file.
-
-This way, you can set a common default editor for your team and provide new contributors with the most suitable editor for your project. You can also use the `che-editor.yaml` file when you need to set a different IDE default for a particular project repository rather than the default IDE of your organization's {prod-short} instance.
+By using the `che-editor.yaml` file, you can set a common default IDE for your team and provide new contributors with the most suitable IDE for your project source code. You can also use the `che-editor.yaml` file when you need to set a different IDE default for a particular project repository rather than the default IDE of your organization's {prod-short} instance.
```

Comment:
```suggestion
Use the `che-editor.yaml` file to set a common default IDE for your team. You can also use the `che-editor.yaml` file to set a different default IDE for a particular project repository rather than the default IDE of your organization's {prod-short} instance.
```

Reply:
I guess the reason I opened for the `by ..., you can`phrasing is because it expresses one of the possibilities rather than the single way of doing this.

It's about about how parallel the two sentences are. In my version, the first sentence is `you can ...` and the second sentence is `you can also ...`.